### PR TITLE
Refactor data-privacy/sensitive-data to data-privacy/records

### PR DESCRIPTION
**Pull request by tjerman**
Please take a look.

### CODE REVIEW

File: compose/rest/handlers/dataPrivacy.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func NewDataPrivacy(h DataPrivacyAPI) *DataPrivacy {
 func (h DataPrivacy) MountRoutes(r chi.Router, middlewares ...func(http.Handler) http.Handler) {
 	r.Group(func(r chi.Router) {
 		r.Use(middlewares...)
-		r.Get("/data-privacy/sensitive-data", h.SensitiveDataList)
+		r.Get("/data-privacy/records", h.RecordList)
```
Comment (darh) [inline compose/rest/handlers/dataPrivacy.go]: both plural or both singular.

**Review by vicpatel — APPROVED** [review]

**Review by darh — APPROVED** [review]
one small fix if you can.

File: compose/service/module.go [hunk 2]
```diff
@@ -444,9 +444,7 @@ func (svc module) FindSensitive(ctx context.Context, filter types.PrivacyModuleF
 		for _, m := range mm {
 			isPrivate := false
 			for _, f := range m.Fields {
-				if !isPrivate {
-					isPrivate = f.Private
-				}
+				isPrivate = isPrivate || f.IsSensitive()
```
Comment (vicpatel) [inline compose/service/module.go:447]: Isn't it always true/false based on the last field in the loop?

Comment (vicpatel) [reply compose/service/module.go:447]: Never mind, My Bad.

**Pull request merged**